### PR TITLE
Create aws-copy-params script

### PR DESCRIPTION
**Pull request by jgondron**
## Create aws-copy-params script

c47b779d635a96a43b2bbbd39d0fb80403a249d8

This adds a script that can be used to easily copy all key value pairs in AWS Parameter Store from one path to another. Can also be used to copy from one region to another.

### CODE REVIEW

**Comment by jgondron** [comment]
Wasn't sure if this was necessary, but we may want to namespace any of our specific aws-* commands, just to differentiate from other aws commands. I did at least add a reference to this project in the help so that a user can tell where this command comes from, so that may be sufficient for now. Am open to opinions on this.

File: bin/aws-copy-params [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+perPage = 10
+withDecryption = True
+
+parser = argparse.ArgumentParser(description='Copies SSM parameters from one path to another. Can currently copy between regions, but not accounts.',
+    epilog='This script is part of the https://github.com/ndlib/commandline-tools project.')
+parser.add_argument('-p','--source-path', help='The source path to copy all parameters from.', required=True)
+parser.add_argument('-l','--source-region', help='The source region to copy all parameters from.', required=True)
+parser.add_argument('-P','--dest-path', help='The destination path to copy all parameters to.', required=True)
+parser.add_argument('-L','--dest-region', help='The destination region to copy all parameters to.', required=True)
+parser.add_argument('-o','--overwrite', help='Force overwrite of existing parameters.', required=False, default=True)
```
Comment (ialford) [inline bin/aws-copy-params]: Do we want to default overwrite parameters? I could see this behavior as a bit dangerous if multiple people are working on a project and aren't communicating re: current state of params.

Comment (jgondron) [reply bin/aws-copy-params]: Probably not. I defaulted this to True atm to make it easy for me to use for my current task. Will change

**Comment by jgondron** [comment]
@ialford 
> one additional question - have you tried to copy parameters that you don't have access to read, and if so, what is the failure behavior?

I have not tested for this specifically. I'm relying on boto3 to handle these errors and print a message accordingly. If we find we need additional error trapping/handling, we can add. There are other gaps here as well, such as decrypt failures, or optionally specifying a different KMS key to decrypt vs the key to use to encrypt the copy etc. Just didn't have a ton of time to dump into this yet.

**Pull request merged**